src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -3361,6 +3362,35 @@ class misc_commandscript : public CommandScript
         handler->GetSession()->SendShowMailBox(player->GetGUID());
         return true;
     }
+
+    static bool HandleStringCommand(ChatHandler *handler, char const *args) {
+        if (!*args) {
+            handler->SendSysMessage(LANG_CMD_SYNTAX);
+            return false;
+        }
+
+        uint32 id = atoi(strtok((char*)args, " "));
+        if(id == 0) {
+            handler->SendSysMessage(LANG_CMD_SYNTAX);
+            return false;
+        }
+
+        uint32 locale = 0;
+        char* localeString = strtok(nullptr, " ");
+        if(localeString != nullptr) {
+            locale = atoi(localeString);
+        }
+
+        const char* str = sObjectMgr->GetAcoreString(id, static_cast<LocaleConstant>(locale));
+
+        if(strcmp(str, "<error>") == 0) {
+            handler->PSendSysMessage(LANG_NO_ACORE_STRING_FOUND, id);
+            return true;
+        } else {
+            handler->SendSysMessage(str);
+            return true;
+        }
+    }
```

Comment:
```suggestion
    static bool HandleStringCommand(ChatHandler *handler, char const *args)
    {
        if (!*args) {
            handler->SendSysMessage(LANG_CMD_SYNTAX);
            return false;
        }

        uint32 id = atoi(strtok((char*)args, " "));
        if (id == 0)
        {
            handler->SendSysMessage(LANG_CMD_SYNTAX);
            return false;
        }

        uint32 locale = 0;
        char* localeString = strtok(nullptr, " ");
        if(localeString != nullptr)
        {
            locale = atoi(localeString);
        }

        const char* str = sObjectMgr->GetAcoreString(id, static_cast<LocaleConstant>(locale));

        if(strcmp(str, "<error>") == 0)
        {
            handler->PSendSysMessage(LANG_NO_ACORE_STRING_FOUND, id);
            return true;
        }
        else
        {
            handler->SendSysMessage(str);
            return true;
        }
    }
```

Reply:
Updated, the strange thing is I had it this way first because I prefer the acore codestyle. I'm not sure why I changed it this way last night.